lua/mapvote/shared/modules/config_schema.lua

```diff
@@ -6,6 +6,7 @@ local schema = SV.Object {
     TimeLimit = SV.Int { min = 1 },
     AllowCurrentMap = SV.Bool(),
     RTVPercentPlayersRequired = SV.Number { min = 0, max = 1 },
+    RTVPercentMulWhenOverpopulated = SV.Number { min = 0, max = 1 },
```

Comment:
Lets go with `RTVPercentPlayersRequiredOverpopulated` and have it not be a mult.

Reply:
Done